core/src/main/resources/xml/xsl/adapter2mermaid.xsl

```diff
@@ -124,19 +126,10 @@
 		<xsl:copy>
 			<xsl:call-template name="defaultPipeCopyActions"/>
 			<!-- Add success forward -->
-			<xsl:element name="forward">
-				<xsl:attribute name="name" select="'success'"/>
-				<xsl:choose>
-					<xsl:when test="exists(../inputWrapper)">
-						<xsl:attribute name="path" select="generate-id(../inputWrapper)"/>
-						<xsl:attribute name="targetID" select="generate-id(../inputWrapper)"/>
-					</xsl:when>
-					<xsl:otherwise>
-						<xsl:attribute name="path" select="$firstPipe/@name"/>
-						<xsl:attribute name="targetID" select="generate-id($firstPipe)"/>
-					</xsl:otherwise>
-				</xsl:choose>
-			</xsl:element>
+			<xsl:call-template name="createForward">
```

Comment:
Moet voor inputValidators niet ook iets gedaan worden met global-forwards?

Reply:
dat wordt standaard gedaan in het defaultPipeCopyActions template. Volgens mij hoeft daar verder niks bijzonders mee te gebeuren. Heb ik dat mis?